### PR TITLE
Create FUNDING.yml

### PR DESCRIPTION
**Pull request by suchov**
Think about creating a funding yml

_Upd_: removed default PR comment contents

### CODE REVIEW

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# These are supported funding model platforms
+
+github: # [suchov] Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
```
Comment (suchov) [inline .github/FUNDING.yml:3]: added my surname just to make the PR valid, we can think about the options that we want to put

**Comment by suchov** [comment]
Just want to suggest us to think about adding the Funding.yml to our most popular repos...

File: .github/FUNDING.yml (hunk 1 above)
Comment (OleksiyRudenko) [inline .github/FUNDING.yml:3]:
```suggestion
github: [suchov] # Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
```

Uncommented a list of names.

How does a user know if they are sponsors-enabled? Anyway, I think one or two would be sufficient

File: .github/FUNDING.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+patreon: # Replace with a single Patreon username
```
Comment (OleksiyRudenko) [inline .github/FUNDING.yml:4]:
```suggestion
patreon: kottans # Replace with a single Patreon username
```

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
LGTM in general. Just need to finalize the lists.

(Boy, that many funding entry point services around. Of those only heard of github, patreon and [opencollective](https://opencollective.com/pricing))

**Review by zonzujiro — APPROVED** [review]

**Pull request merged**